test/unit/vdom/properties.js

```diff
@@ -38,13 +38,21 @@ describe('vdom/properties', () => {
 
   it('false should remove the attribute', done => {
```

Comment:
Agreed that we should explicitly have an answer to this, but not sure what the expected behaviour is/should be...